.github/labels.json

```diff
@@ -0,0 +1,57 @@
+[
+	{ "name": "💩 template incomplete", "color": "#4E342E" },
+	{ "name": "💩 template removed", "color": "#4E342E" },
+
+	{ "name": "b¹ 🐞 code-splitting", "color": "#F44336" },
+	{ "name": "b² 🐞 scope-hoisting", "color": "#F44336" },
+	{ "name": "b³ 🐞 tree-shaking", "color": "#F44336" },
+
+	{ "name": "c¹ ⋅ discussion", "color": "#1976D2" },
+	{ "name": "c² ⋅ feedback wanted", "color": "#F9A825" },
+	{ "name": "c³ ⋅ PR welcome", "color": "#1B5E20" },
+	{ "name": "c⁴ ⋅ need more info", "color": "#6A1B9A" },
+	{ "name": "c⁵ ⋅ question", "color": "#C2185B" },
+	{ "name": "c⁶ ⋅ request for comments", "color": "#BBDEFB" },
+
+	{ "name": "p¹ ⋅ electron", "color": "#B2DFDB" },
```

Comment:
I'm not 100% sure but I would not expect `electron` to be an important environment. `browser`, on the other hand, would be useful here.

Reply:
True. I'll change that up.